oms-hypervReplica-solution/azuredeploy.json

```diff
@@ -0,0 +1,650 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.1.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of your OMS Log Analytics workspace."
+      }
+    },
+    "workspaceRegionId": {
+      "type": "string",
+      "allowedValues": [
+        "East US",
+        "West Europe",
+        "Southeast Asia",
+        "Australia Southeast",
+        "West Central US"
+      ],
+      "metadata": {
+        "description": "The region of your OMS Log Analytics workspace."
+      },
+      "defaultValue": "West Europe"
+    },
+    "accountName": {
```

Comment:
thanks @bentaylorwork - I know this might be confusing, but those parameters are recommended for solutions going to marketplace. For GitHub (here), I suggest you pivot to "automationName" or "automationAccountName"